src/main/java/com/mongodb/shardsync/command/sync/SyncShardingCommand.java

```diff
@@ -0,0 +1,29 @@
+package com.mongodb.shardsync.command.sync;
+
+import com.mongodb.shardsync.ShardConfigSync;
+import com.mongodb.shardsync.SyncConfiguration;
+
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+import java.util.concurrent.Callable;
+
+@Command(name = "sharding", 
+         aliases = {"shard-collections"},
+         description = "Shard collections on destination to match source")
```

Comment:
Is this a subset of the metadata-sync command?

Reply:
This would be for a different use of SCS other than migrations. Although this kind of use is not common, it is a handy tool to have as are many of the options SCS has that are of general utility -- sync sharding, sync indexes, etc.